gbpservice/neutron/tests/unit/nfp/configurator/api/v1/controllers/test_controller.py

```diff
@@ -10,18 +10,22 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-#import json
-import mock
-import unittest
 
+import mock
 import oslo_serialization.jsonutils as jsonutils
 import pecan
-from pecan import rest
+PECAN_CONFIG_FILE = '/opt/stack/gbp/gbpservice/nfp/configurator/api/config.py'
```

Comment:
Assumed test cases will be run under /opt/stack/gbp/. If it is kept in other location then will have to modify this variable. any other suggestion ?